### PR TITLE
Make systemd guide clearer on obtaining username and python path

### LINKED ISSUE #3462: [Docs] auto-restart using systemd  can be misleading
Currently the docs say the following:
Paste the following and replace all instances of `username` with the username, and `path` with the location you obtained above:

However in the section mentioned only the `path` is mentioned.

We could have a command in the section above to get the user `echo $USER` or `whoami` and explicitely mention that is the desired output for `username`  and that the outputs of `which python` is the desired output for `path`

### PR DESCRIPTION
**Pull request by Zephyrkul**
### Type

- [X] Docs

### Description of the changes
Resolves #3462

### CODE REVIEW

File: docs/autostart_systemd.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ Then create the new service file:
 
 :code:`sudo -e /etc/systemd/system/red@.service`
 
-Paste the following and replace all instances of :code:`username` with the username, and :code:`path` with the location you obtained above:
+Paste the following and replace all instances of :code:`username` with your Linux username (you can use the :code:`whoami` command if you're unsure), and :code:`path` with the location you obtained above:
```
Comment (Jackenmen) [inline docs/autostart_systemd.rst]: This is imo a little unclear on *why* would user want to use this command, I think adding something like "if you're unsure what your username is" would be enough here, but anything that makes it more clear when you should use the command and/or what is the reader "unsure" of is gonna work here so if you think something would fit better here, you can of course put that instead ;)
```suggestion
Paste the following and replace all instances of :code:`username` with your Linux username (you can use the :code:`whoami` command if you're unsure what your username is), and :code:`path` with the location you obtained above:
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]
One change on clarity needed

File: docs/autostart_systemd.rst (hunk 1 above)
Comment (Zephyrkul) [reply docs/autostart_systemd.rst]: I decided to rewrite that section entirely since the whole thing was a little unclear at a couple points.

**Review by Jackenmen — APPROVED** [review]
Thanks for the PR

**Pull request merged**